client/extensions/woocommerce/app/settings/taxes/taxes-taxjar.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionHeader from 'woocommerce/components/action-header';
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import Main from 'components/main';
+import SettingsNavigation from '../navigation';
+
+class SettingsTaxesTaxJar extends Component {
+	static propTypes = {
+		className: PropTypes.string,
+		site: PropTypes.shape( {
+			options: PropTypes.shape( {
+				admin_url: PropTypes.string.isRequired,
+			} ),
+		} ).isRequired,
+	};
+
+	render = () => {
+		const { className, site, translate } = this.props;
+
+		const breadcrumbs = [
+			<a href={ getLink( '/store/settings/:site/', site ) }>{ translate( 'Settings' ) }</a>,
+			<span>{ translate( 'Taxes' ) }</span>,
+		];
+
+		const adminUrl =
+			site.options.admin_url +
+			'admin.php?page=wc-settings&tab=integration&section=taxjar-integration';
+		const pluginUrl = getLink( '/plugins/taxjar-simplified-taxes-for-woocommerce/:site', site );
+
+		return (
+			<Main className={ classNames( 'settings-taxes', className ) }>
+				<ActionHeader breadcrumbs={ breadcrumbs } />
+				<SettingsNavigation activeSection="taxes" />
+				<div className="taxes__tax-jar">
+					<Card>
+						<p>
+							{ translate(
+								"Your store's taxes are being managed by the {{b}}TaxJar - Sales Tax Automation " +
+									'for WooCommerce{{/b}} plugin. This optional plugin provides features such as:',
+								{
+									components: {
+										b: <strong />,
+									},
+								}
+							) }
+						</p>
+						<ul>
+							<li>{ translate( 'Automated reporting & filing' ) }</li>
+							<li>{ translate( 'Multi-nexus support' ) }</li>
+						</ul>
+						<p>
+							{ translate(
+								"If you don't need these features we recommend {{a}}removing this plugin{{/a}}. Don't worry, " +
+									"{{b}}we'll still handle sales tax calculations for you{{/b}}.",
+								{
+									components: {
+										a: <a href={ pluginUrl } className="taxes__tax-jar-deactivate" />,
+										b: <strong />,
+									},
+								}
+							) }
+						</p>
+						<p>
+							{ translate( "You can manage this plugin's settings in wp-admin. " ) }
+							<ExternalLink icon href={ adminUrl } rel="noopener noreferrer">
+								{ translate( 'Plugin settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Edit plugin settings' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).